doc/man7/provider.pod

```diff
@@ -147,6 +147,30 @@ The number for this operation is B<OSSL_OP_KEYEXCH>.
 The functions the provider can offer are described in
 L<provider-keyexch(7)>
 
+=item Asymmetric Ciphers
+
+In the OpenSSL libraries, the corresponding method object is
+B<EVP_ASYM_CIPHER>.
```

Comment:
In some manuals, for example `doc/man3/EVP_MAC.pod`, the type itself is one of the names of that manual.  The idea behind that is that if you want to know something about a type, you will be able to do `man EVP_MAC` and get at least a starting page.  If we did that consistently, `B<EVP_ASYM_CIPHER>` could be changed to `L<EVP_ASYM_CIPHER(3)>`, permitting easy links.

I would see that as an advantage, what are your thoughts?

Reply:
Yes - I think it would be great if we had that.

Reply:
It's really just a matter of adding those names in the proper files.  I would suggest that they can be added in the corresponding *TYPE*_new.pod, and that file can be renamed to *TYPE*.pod.

Reply:
Strong suggestion to do "make doc-nits cmd-nits" before you push :)